docs-js/release-notes.mdx

```diff
@@ -0,0 +1,14 @@
+---
+id: release-notes
+title: Release Notes
+sidebar_label: Release Notes
+description: Release notes of SAP Cloud SDK for AI, stay up to date with the recent features, fixes, dependency updates and recommendations.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+  - release notes
+---
+
+Work in progress, please check back later.
```

Comment:
[q] Why do we have 2 release-notes files?

Reply:
The other one was copy pasted. Removed and should be handled by @KavithaSiva 's PR